### ISSUE TITLE
LandingPage Incorrect Content Order 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#780]
As mentioned in the previous “Missing Heading Hierarchy” section, several pieces of text are present on the Landing Page which are visually styled to look like headings.  In many such cases, these headings are preceded by a related image (such as the collection of logos before the “LMS Integration” feature heading). 
 
The current lack of semantic heading mark-up aside, this placement is problematic for screen reader users because the corresponding heading appears in the DOM after the image instead of before it.  Once proper heading mark-up has been implemented, users will be unsure of the relationship between the image and its corresponding heading, for example by: 
 
• assuming that an image is actually part of the previous headed section; or 
• skipping past the image altogether while navigating through the page by heading. 
 
Suggested Solution 
 
Headings must always be placed in the DOM before the content which they describe.  Here, the < div> elements with the className attribute of “{styles.imageSection}” should be swapped with those which have the className “{ styles.header}”, so that the header is encountered first.

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/dd67712d907c33c9b1c321fe3488977c8422267f.
